src/runtime.js

```diff
@@ -513,6 +514,43 @@ export class MultidocManager {
       amp.resources = Services.resourcesForDoc(ampdoc);
     }
 
+    /**
+     * Expose amp-bind getState
+     * @param {string} name - Name of state or deep state
+     * @return {Promise<*>} - Resolves to a copy of the value of a state
+     */
+    amp['getState'] = name => {
+      return Services.bindForDocOrNull(shadowRoot).then(bind => {
+        if (!bind) {
+          return Promise.reject('amp-bind is not available in this document');
+        }
+        return bind.getState(name);
+      });
+    };
+
+    /**
+     * Expose amp-bind setState
+     * @param {(!JsonObject|string)} state - State to be set
+     * @return {Promise} - Resolves after state and history have been updated
+     */
+    amp['setState'] = state => {
+      return Services.bindForDocOrNull(shadowRoot).then(bind => {
+        if (!bind) {
+          return Promise.reject('amp-bind is not available in this document');
+        }
+        if (typeof state === 'string') {
+          // Emulate UIEvent 'click'
+          return bind.setStateWithExpression(
+            /** @type {string} */ (state),
+            /** @type {!JsonObject} */ ({event: 1})
```

Comment:
Is `{event: 1}` useful here?

Reply:
I haven't dug too deep into how the scope parameter is used, but at least an empty object needs to be passed. I opted to mimic the object that is passed by a single click event here.

Reply:
Empty object will suffice then. :)

It's used for things like `on="submit-success: AMP.setState({foo: event.response})"`, but there's no real "event context" in this case.